tools/github_actions_install.sh

```diff
@@ -11,7 +11,7 @@ elif [[ "$SPHINX_VERSION" == "dev" ]]; then
 elif [[ "$SPHINX_VERSION" == "old" ]]; then
     SPHINX_INSTALL="sphinx[test]==6.1.0"
 else  # for the "build site" jobs
-    SPHINX_INSTALL='"sphinx<7.3"'
+    SPHINX_INSTALL="sphinx<7.3"
```

Comment:
hah, I guessed wrong. I expected the double-quotes to get stripped out and the `<` misinterpreted by bash, but looks like that's not happening. 🙏🏻